js/src/ui/Container/container.css

```diff
@@ -48,7 +51,8 @@ $transitionAll: all 0.75s cubic-bezier(0.23, 1, 0.32, 1);
 
     .hoverOverlay {
       background: $backgroundHover;
-      transform: scale(1, 1);
+      /*transform: scale(1, 1);*/
```

Comment:
I left it in on purpose - want to revisit the actual animations, so need guidance as to what it was. (There are a couple)